oxy/yaml/bms.yaml

```diff
@@ -0,0 +1,243 @@
+!Messages
```

Comment:
Nit: can we rename the `yaml` directory to something like `msgs` or something else that describes what is contained in the yaml files?